src/components/combobox/combobox.tsx

```diff
@@ -1147,21 +1149,25 @@ export class Combobox
     const single = this.selectionMode === "single";
 
     return (
-      <Host onKeyDown={this.keydownHandler}>
+      <Host>
         <div
           aria-autocomplete="list"
+          aria-controls={`${listboxUidPrefix}${guid}`}
           aria-expanded={toAriaBoolean(open || active)}
           aria-haspopup="listbox"
           aria-labelledby={`${labelUidPrefix}${guid}`}
+          aria-live="polite"
           aria-owns={`${listboxUidPrefix}${guid}`}
           class={{
             wrapper: true,
             "wrapper--single": single || !this.selectedItems.length,
             "wrapper--active": open || active
           }}
           onClick={this.clickHandler}
+          onKeyDown={this.keydownHandler}
           ref={this.setReferenceEl}
           role="combobox"
+          tabindex="0"
```

Comment:
Should we do something like `tabIndex={!this.disabled ? 0 : null}`

Reply:
I think it's fine since `updateHostInteraction` takes care of disabling tabbing into the component.

Reply:
Great catch! 🐟 Will incorporate in.